src/createTheme.spec.ts

```diff
@@ -16,7 +17,7 @@ describe('createTheme', () => {
         ...defaultThemeColors,
         primaryDark: 'foo',
```

Comment:
@erikshestopal  I dont get an error if typed the key wrong, `primaryDark` -> `priimaryDark`

Reply:
<img width="1223" alt="Screen Shot 2020-07-27 at 12 20 24 PM" src="https://user-images.githubusercontent.com/6404663/88582442-95819c00-d003-11ea-8265-e290e27cec79.png">

Reply:
Or are you saying if you add the typo to the override `colors` object?

Reply:
I meant online 18

Reply:
@vietnogigoodrx do you think you can fix that issue? I hacked this over the weekend and didn't address small edge cases. I have some other priorities that I have to work on but would love to help sometime later this week.